### ISSUE TITLE
Some rst sections' content omitted from output

### ISSUE THREAD

**Issue by AndydeCleyre** [Textualize/rich-cli#31]
```console
$ git clone https://github.com/pypa/setuptools
$ rich setuptools/docs/pkg_resources.rst
```

The last bunch of sections have headers but no content in the output, though content exists in the input.

![image](https://user-images.githubusercontent.com/1787385/157117758-8370a0b4-96e4-4880-a068-43a91c049e6c.png)

**Comment by willmcgugan** [comment]
@wasi_master Do you recognise this issue?

**Comment by wasi-master** [comment]
Those may just be footers? And footers are usually in the bottom. I'll have to test further. But I'll need a reproducible example

**Comment by wasi-master** [comment]
I've fixed the issue locally

![Image 1](https://user-images.githubusercontent.com/63045920/157163581-369cd029-93a2-4c3e-a395-5afbbfa6c7e5.png)

**Comment by wasi-master** [comment]
@willmcgugan Fixed in 1.1.7, You can close this issue
